### ISSUE TITLE
Home page milestones

### ISSUE THREAD

**Issue by hoangviet1993** [oppia/foundation-website#8]
- [x] Initial commit to setup folder and directive.
- [x] Basic page layout.
- [x] Obtain needed assets ( photos, icons and svgs ) from designers. 
- [x] (Aug 2018) Rachel feedback: Misleading headline.
- [x] (Aug 2018) Rachel feedback: Insert video into laptop screen
- [x] Optimize video loading (two separate video tag loading the same video twice)

**Comment by hoangviet1993** [comment]
Hi @seanlip 
I have been trying to wrap my head around this duplicated video loading issue:
![screen shot 2018-10-12 at 10 44 12 pm](https://user-images.githubusercontent.com/11153258/46900883-b4875c00-ce77-11e8-9ae3-eecfcc802f8e.png)
Context:
For the home page (home.html), there are 2 separate HTML 5 video tag (`<video>`). One is for mobile and tablet viewport (<960px) and one is for desktop viewport (>960px).

I needed to use 2 separate video tags because of the differences between mobile view vs desktop mock. For mobile, the page is supposed to be, from top-bottom, picture-text-picture-text...

If I was to use only 1 video tag, the laptop would have to go under the text "A new approach to online learning". Or, the desktop would need to be on the left side instead of on the right side currently.

I have tried separating the HTML codes of the `<video>` tag into its own directive template. It did not work which is why I am here asking for opinions on how to solve this.

A [SO post](https://stackoverflow.com/a/24532111) advised to use `<canvas>`, which basically re-draw the video as a series of pictures(not sure if my read is correct here). Again, I am not sure if this is the desired effect on mobile but I can certainly do that.
